stdpopsim/genomes.py

```diff
@@ -386,6 +386,70 @@ def add_dfe(self, intervals, DFE):
         self.dfe_list.append(DFE)
         self.interval_list.append(intervals)
 
+    def add_single_site_mutation_type(
+        self, id, selection_coeff=0.0, dominance_coeff=1.0
+    ):
+        """
+        Adds a DFE representing a mutation at a single site, contain a single
```

Comment:
It's a bit confusing to call this a DFE ("distribution of fitness effects"), even though that's what it is under the hood. I'm not sure how else to explain this right now, though?